ui/App/SharedComponents/PeerSelector.svelte

```diff
@@ -157,48 +159,52 @@
     data-cy="peer-selector"
     hidden={expanded}
     on:click|stopPropagation={show}>
-    <Peer peer={selected} />
+    <UserIdentity
+      boldHandle={true}
+      urn={selected.identity.urn}
+      handle={selected.identity.metadata.handle}
+      disableHovercard={true}>
+      <PeerBadge user={selected} />
+    </UserIdentity>
     <div class="selector-expand">
       <Icon.ChevronUpDown
         style="vertical-align: bottom; fill: var(--color-foreground-level-4)" />
     </div>
   </div>
   <div class="peer-dropdown-container" data-cy="peer-dropdown-container">
     <div
-      bind:clientHeight={dropdownHeight}
       class="peer-dropdown"
       hidden={!expanded}
       class:rounded={standalone}
-      style={`border-bottom-right-radius: ${
-        dropdownHeight > 40 ? "0.5rem" : "0"
-      }`}>
+      class:rounded-bottom-right={!standalone && peers.length > 1}>
       {#each orderPeers(peers) as peer (peer.peerId)}
         <div
           data-cy="peer-dropdown-entry"
-          class="entry"
+          class="entry rounded-left"
+          class:rounded-right={standalone}
+          class:rounded-bottom-right={!standalone && peers.length > 1}
```

Comment:
I find it hard to understand how rounded corners are determined. On particular issue is that the criteria based on state (`standalone`) are in a different location from the criteria based on position (`:first-of-type`). I’d suggest the following:
```diff
diff --git a/ui/App/SharedComponents/PeerSelector.svelte b/ui/App/SharedComponents/PeerSelector.svelte
index 3ca7ebf6..ffe5bb87 100644
--- a/ui/App/SharedComponents/PeerSelector.svelte
+++ b/ui/App/SharedComponents/PeerSelector.svelte
@@ -115,24 +115,19 @@
     padding: 0 0.5em;
   }
 
-  .rounded-right:first-of-type {
+  .entry:first-child {
     border-top-right-radius: 0.5rem;
-  }
-
-  .rounded-right:last-of-type {
-    border-bottom-right-radius: 0.5rem;
-  }
-
-  .rounded-left:first-of-type {
     border-top-left-radius: 0.5rem;
   }
 
-  .rounded-left:last-of-type {
+  .entry:last-child {
+    border-bottom-right-radius: 0.5rem;
     border-bottom-left-radius: 0.5rem;
   }
 
-  .rounded-bottom-right:last-of-type {
-    border-bottom-right-radius: 0.5rem;
+  .entry:not(.standalone):first-child {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
   }
 
   .entry.enabled {
@@ -180,9 +175,8 @@
       {#each orderPeers(peers) as peer (peer.peerId)}
         <div
           data-cy="peer-dropdown-entry"
-          class="entry rounded-left"
-          class:rounded-right={standalone}
-          class:rounded-bottom-right={!standalone && peers.length > 1}
+          class="entry"
+          class:standalone
           class:enabled={peer.role !== PeerRole.Tracker}
           class:selected={peer.identity.peerId === selected.identity.peerId}
           on:click|stopPropagation={() => onSelect(peer)}>
```

Reply:
Hm, yeah, I think the component is a bit too complex at the moment. And maybe the best would be to roll back all the rounding related changes because we opted of not showing the hover cards in the dropdown?

The problem with your proposed solution is that when there are multiple entries in the dropdown, the bottom right corner isn't rounded.

 <img width="530" alt="Screenshot 2021-10-21 at 10 33 32" src="https://user-images.githubusercontent.com/158411/138242056-56e8e911-05dc-4769-91a3-d1e2665fc996.png">

For reference, here are all the use cases:
### With "edit remotes" button on the right side
#### Single entry


<img width="558" alt="Screenshot 2021-10-21 at 10 33 16" src="https://user-images.githubusercontent.com/158411/138243246-14fb01b5-3f2c-4401-90ec-97ef3e0724e9.png">


#### Multiple entries (note the rounded bottom right corner)
<img width="552" alt="Screenshot 2021-10-21 at 10 34 14" src="https://user-images.githubusercontent.com/158411/138242066-95c7d383-99a3-4016-a499-3abde65fd6f5.png">

### Standalone (no "edit remotes" button on the right side)
#### Single entry
<img width="519" alt="Screenshot 2021-10-21 at 10 33 02" src="https://user-images.githubusercontent.com/158411/138242325-5594f65b-98e7-41b8-a774-2054903dd37e.png">

#### Multiple entries
<img width="511" alt="Screenshot 2021-10-21 at 10 32 53" src="https://user-images.githubusercontent.com/158411/138242312-ff83a0a6-64e6-4256-b940-037896244c82.png">